src/utils/styles.ts

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+export const layoutStyles = StyleSheet.create({
+  ColumnJustifyStart: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+  },
+  RowAlignCenter: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  noPaddings: { paddingLeft: 0, paddingRight: 0 },
```

Comment:
this is too generic a name to contain only `paddingLeft` and `paddingRight`. From such a name, I would expect `{ padding: 0 }`

Reply:
Thanks. Commits 044c8f9 https://github.com/urbi-mobility/urbi-react-native/pull/233/commits/4a5bba8bb53655f4051b794c04e859f5dece4ac2